org.eclipse.xtext.ide/src/org/eclipse/xtext/ide/server/semantictokens/SemanticTokensService.java

```diff
@@ -54,13 +55,42 @@ protected SemanticTokensService() {
 	}
 
 	protected void addTokenTypes() {
-		tokenTypes.add(HighlightingStyles.KEYWORD_ID);
-		tokenTypes.add(HighlightingStyles.DEFAULT_ID);
-		tokenTypes.add(HighlightingStyles.NUMBER_ID);
+		tokenTypes.add(SemanticTokenTypes.Namespace);
```

Comment:
Thank you for completing the list.
I've to say I don't quite understand this protected method - given that the private field tokenTypes is not accessible, I don't see how a client might override it. Can this be private?

Reply:
Ok, I missed to getter during the review. Forget the comment.